trade-finance-manager-ui/component-tests/utilisation-reports/add-payment.component-test.ts

```diff
@@ -1,10 +1,27 @@
-const { pageRenderer } = require('../pageRenderer');
-const { anAddPaymentViewModel, aRecordedPaymentDetailsViewModel } = require('../../test-helpers/test-data/view-models');
+import { CURRENCY, FEE_RECORD_STATUS } from '@ukef/dtfs2-common';
+import { pageRenderer } from '../pageRenderer';
+import { anAddPaymentViewModel, aRecordedPaymentDetailsViewModel } from '../../test-helpers/test-data/view-models';
 
 const page = '../templates/utilisation-reports/add-payment.njk';
 const render = pageRenderer(page);
 
 describe(page, () => {
+  it('should add error prefix to page title when there are errors', () => {
+    const addPaymentViewModel = anAddPaymentViewModel();
+    addPaymentViewModel.errors.errorSummary = [{ text: 'an error', href: 'error-href' }];
+    const wrapper = render(addPaymentViewModel);
+
+    wrapper.expectPageTitle().toRead('Error - Add a payment');
+  });
+
+  it('should not add error prefix to page title when there are no errors', () => {
+    const addPaymentViewModel = anAddPaymentViewModel();
+    addPaymentViewModel.errors.errorSummary = [];
+    const wrapper = render(addPaymentViewModel);
+
+    wrapper.expectPageTitle().toRead('Add a payment');
+  });
+
```

Comment:
(just as a knowledge share, resolve once read, no action needed)

We could have these as `withErrorPrefixInPageTitleTests.ts`
```
export const withErrorPrefixInPageTitleTests = ({aViewModel, defaultPageTitle}) => {
  it('should add error prefix to page title when there are errors', () => {
    const viewModel = aViewModel();
    viewModel.errors.errorSummary = [{ text: 'an error', href: 'error-href' }];
    const wrapper = render(viewModel);

    wrapper.expectPageTitle().toRead(`Error - ${defaultPageTitle}`);
  });

  it('should not add error prefix to page title when there are no errors', () => {
    const viewModel = aViewModel();
    viewModel.errors.errorSummary = [];
    const wrapper = render(viewModel);

    wrapper.expectPageTitle().toRead(defaultPageTitle);
  });
}
``` 

and then for these tests, we wouldnt have to repeat this and we could just have
```
withErrorPrefixInPageTitleTests({
    aViewModel: anAddPaymentViewModel,
    defaultPageTitle: 'Add a payment'
})
```
in our 4 test files -- removing a fair chunk of repeated tests